README.md

```diff
@@ -1,32 +1,79 @@
+|AWS_LABELER_DENIED_ACCOUNT_IDS|`--denied-account-ids` `-d`|No|123456789102,123456789103 (default: `None`)|A list of account IDs that should NOT be scored. All accounts will be scored EXCEPT accounts in this list. This variable is mutually exclusive with ALLOWED_ACCOUNT_IDS|
+|AWS_LABELER_ALLOWED_REGIONS|`--allowed-regions` `-ar`|No|eu-west-1,eu-central-1 (default: `None`)|A list of regions that should be included. No regions will be included EXCEPT for regions in this list. This variable is mutually exclusive with DENIED_REGIONS|
+|AWS_LABELER_DENIED_REGIONS|`--denied-regions` `-dr`|No|eu-west-1,eu-central-1 (default: `None`)|A list of regionss that should NOT be included. All regions will be included EXCEPT regions in this list. This variable is mutually exclusive with ALLOWED_REGIONS|
+|AWS_LABELER_EXPORT_PATH|`--export-path` `-p`|Yes if `export metrics` or `export all` is true|/tmp/aws_output (default: `None`)|The location where the output can be stored.|
+|AWS_LABELER_EXPORT_METRICS|`--export-metrics` `-em`|No|`False` (default)|Exports metrics/statistics without sensitive findings data in JSON formatted files to the specified directory or S3 location.|
+|AWS_LABELER_EXPORT_ALL|`--export-all` `-ea`|No|`True` (default)|Exports metrics/statistics along with sensitive findings data in JSON formatted files to the specified directory or S3 location.|
+|AWS_LABELER_TO_JSON|`--to-json` `-j`|No|`False` (default)|Return the report in json format.|
+
+## Permissions
+Since the energylabeler can run on either a full Landing Zone, or a single account, different permissions are required.
+
+### Landing zone
+The following permissions are required to run on a landing zone.
 ```
+"organizations:DescribeOrganization",
+"organizations:ListAccounts",
+"organizations:DescribeAccount",
+"iam:ListAccountAliases",
+"ec2:DescribeRegions",
+"securityhub:ListFindingAggregators",
+"securityhub:GetFindings"
+```
+A full example of a policy can be found in the file [EnergyLabelerPolicy_example.json](policy_examples/EnergyLabelerPolicy_example.json)
+
+### Single account
+The following permissions are required to run on a single account.
+```
+"iam:ListAccountAliases",
+"ec2:DescribeRegions",
+"securityhub:ListFindingAggregators",
+"securityhub:GetFindings"
+```
+
+A full example of a policy can be found in the file [EnergyLabelerPolicySingle_example.json](policy_examples/EnergyLabelerPolicySingle_example.json)
 
-## Container signature
-All container in this repository are signed and their signature can be verified against the following public key:
+## Examples
+
+Running the container for your local machine (assuming you already setup `~/.aws` folder with your credentials and config)
 
-#### **cosign.pub**
```

Comment:
Please leave this either as a file in the repo or in the README to verify the container signature